### PR TITLE
Add different emotions on loading screen

### PR DESCRIPTION
**Pull request by judge2020**
has been tested and works 100%
Should be easy adding in new emotions, just define a dictionary (above the list!) with the emote name and frame it should use, then add that dictionary to emotelist
the album of these new emotions can be found [here](http://imgur.com/a/ldu1D)

### CODE REVIEW

**Comment by ToontownLittleCat** [comment]
At least the pull request isn't from TTF for once. xD

File: toontown/toontowngui/ToontownLoadingScreen.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (misread) [inline toontown/toontowngui/ToontownLoadingScreen.py]: There's a random new line here.

**Comment by judge2020** [comment]
Fixed

**Comment by misread** [comment]
yay judge finally did something useful for the tti team.

I'll still wait to see what Chan thinks about it.

**Comment by judge2020** [comment]
It took 1 hour to find the right frame for these 3 emotions, could there be a tool to view the bam files frame-by-frame?

**Comment by forkrave** [comment]
Or you could just have the animations play instead of just having one
frame.

**Comment by judge2020** [comment]
I'm pretty sure this would crash the client due to the loading screen not supporting playing the animations, but I'll test that tonight

**Comment by forkrave** [comment]
No, I mean like have them play in separate frames, giving the player an
illusion the animation is playing. ;)

**Comment by judge2020** [comment]
Plus most people have an fast enough computer to where the playground will load before the animation plays through

**Comment by forkrave** [comment]
Well, there are those 4 GBs of RAM people. :P Plus the loading screen to
the other loading screen is kind of slow.

**Comment by judge2020** [comment]
the album of these new emotions can be found [here](http://imgur.com/a/ldu1D)

**Comment by forkrave** [comment]
Looks good :)

**Comment by judge2020** [comment]
@forkrave thanks :)

**Comment by gamesawdrew** [comment]
You are able to play live animations through the loading screen, just set the beginning frame to 0 and the end to whatever the last frame # is

**Comment by mgracer48** [comment]
toons can do pushups? LOL

**Comment by judge2020** [comment]
@mgracer48 it's the down animation that goes to the party dance floor

**Comment by mgracer48** [comment]
oh lol

**Comment by misread** [comment]
I'd like to see where this goes.

**Comment by judge2020** [comment]
it's a great feature

**Comment by misread** [comment]
I guess this won't be happening. Closing.

**Pull request closed without merging**